addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1428,9 +1437,16 @@ MockServer.include({
             notifications = this._mockMailNotification_NotificationFormat(
                 notifications.map(notification => notification.id)
             );
-            const trackingValueIds = this._getRecords('mail.tracking.value', [
+            let trackingValueIds = this._getRecords('mail.tracking.value', [
                 ['id', 'in', message.tracking_value_ids],
             ]);
+            trackingValueIds = trackingValueIds.map(tracking => ({
```

Comment:
keep const and make a new variable, one is the records, and the other is the format result